build.override.targets

```diff
@@ -3,37 +3,39 @@
 
   <PropertyGroup>
     <TraversalBuildDependsOn>
+      WcfCopyProjectJson;
       WcfSetup;
       $(TraversalBuildDependsOn);
       WcfCleanup;
     </TraversalBuildDependsOn>
   </PropertyGroup>
-  
+
   <PropertyGroup>
     <TraversalCleanDependsOn>
       WcfClean;
       $(TraversalCleanDependsOn);
     </TraversalCleanDependsOn>
   </PropertyGroup>
-  
+
   <PropertyGroup>
     <WcfSelfHostSetupLog>$(MSBuildThisFileDirectory)\SelfHostedWcfServiceSetup.log</WcfSelfHostSetupLog>
     <WcfSelfHostCleanupLog>$(MSBuildThisFileDirectory)\SelfHostedWcfServiceCleanup.log</WcfSelfHostCleanupLog>
+    <ScenarioLatestDependencyJson>$(MSBuildThisFileDirectory)\src\System.Private.ServiceModel\tools\test\JsonFiles\stable.dependencies.json</ScenarioLatestDependencyJson>
+    <UseStableDependencies Condition="'$(UseStableDependencies)'==''">true</UseStableDependencies>
```

Comment:
I am leaning towards flipping the bit here. While we are progressing the development, we will have more and more non-stable dependencies so `UseStableDependencies` will really become `UseSpaghettiDependencies` 😄, which can be confusing to people. If we call the knob `UseLatestDependencies`, it will stay true (and we set it here default to false).